lib/ansible/modules/windows/win_package.py

```diff
@@ -77,23 +77,22 @@
 '''
 
 EXAMPLES = '''
-# Playbook example
-- name: Install the vc thingy
+- name: Install the vVisual C thingy
   win_package:
-    name="Microsoft Visual C thingy"
-    path="http://download.microsoft.com/download/1/6/B/16B06F60-3B20-4FF2-B699-5E9B7962F9AE/VSU_4/vcredist_x64.exe"
-    Product_Id="{CF2BEA3C-26EA-32F8-AA9B-331F7E34BA97}"
-    Arguments="/install /passive /norestart"
+    name: Microsoft Visual C thingy
+    path: http://download.microsoft.com/download/1/6/B/16B06F60-3B20-4FF2-B699-5E9B7962F9AE/VSU_4/vcredist_x64.exe
+    Product_Id: {CF2BEA3C-26EA-32F8-AA9B-331F7E34BA97}
```

Comment:
These will need to remain quoted (either kind)- naked `{` at the beginning of a YAML value will be interpreted as a dictionary.